src/client/client.js

```diff
@@ -249,6 +249,15 @@ export default class Client {
     }
 
     // Post routes
+    fetchPosts = (channelId, onRequest, onSuccess, onFailure) => {
+        return this.doFetch(
+            `${this.getPostsRoute(channelId)}/page/0/60`,
```

Comment:
When we implement pagination of posts we'll pass in the `page` and `offset` params so this function is more like `Client.getPostsPage` in the webapp.